_api-reference/ingest-apis/index.md

```diff
@@ -9,6 +9,8 @@ redirect_from:
 
 # Ingest APIs
 
-Before you index your data, OpenSearch's ingest APIs help transform your data by creating and managing ingest pipelines. Pipelines consist of **processors**, customizable tasks that run in the order they appear in the request body. The transformed data appears in your index after each of the processor completes.
+Ingest APIs help you efficently transform and process data for an index by creating pipelines for dat ingestion in OpenSearch. Ingest pipelines use **processors**, customizable tasks that run in the order that they appear in the request body (that is, the data sent to the server as part of the HTTP request to ingest data in an API). Once the data is transformed, processed, and indexed, it is stored in OpenSearch and ready for you to search.
```

Comment:
As vale points out, typo: efficiently
"... creating pipelines for data ingestion ..." ?